Python/Product/PythonTools/ptvsd_launcher.py

```diff
@@ -28,42 +28,31 @@
 import sys
 import traceback
 
-try:
-    ptvs_lib_path = os.path.dirname(__file__)
-    sys.path.insert(0, ptvs_lib_path)
-    import ptvsd.debugger as vspd
-except:
-    traceback.print_exc()
-    print('''
-Internal error detected. Please copy the above traceback and report at
-https://go.microsoft.com/fwlink/?LinkId=293415
-
-Press Enter to close. . .''')
-    try:
-        raw_input()
-    except NameError:
-        input()
-    sys.exit(1)
-finally:
-    sys.path.remove(ptvs_lib_path)
-
 # Arguments are:
 # 1. Working directory.
 # 2. VS debugger port to connect to.
 # 3. GUID for the debug session.
 # 4. Debug options (as list of names - see enum PythonDebugOptions).
-# 5. '-m' or '-c' to override the default run-as mode. [optional]
-# 6. Startup script name.
-# 7. Script arguments.
+# 5. '-g' to use the installed ptvsd package, rather than bundled one.
+# 6. '-m' or '-c' to override the default run-as mode. [optional]
+# 7. Startup script name.
+# 8. Script arguments.
 
 # change to directory we expected to start from
 os.chdir(sys.argv[1])
 
 port_num = int(sys.argv[2])
 debug_id = sys.argv[3]
-debug_options = vspd.parse_debug_options(sys.argv[4])
+debug_options = set([opt.strip() for opt in sys.argv[4].split(',')])
+
 del sys.argv[0:5]
 
+# Use bundled ptvsd or not?
+bundled_ptvsd = True
+if sys.argv and sys.argv[0] == '-g':
```

Comment:
@karthiknadig - this is the option that you'll need to give to the launcher when experimental flag is checked.